server/graphql/invoice/src/mutations/customer_return/insert.rs

```diff
@@ -157,6 +158,7 @@ impl CustomerReturnLineInput {
             expiry_date,
             batch,
             pack_size,
+            stock_line_id: None,
```

Comment:
We'll continue to create new stock lines from existing customer return functionality, but provision is now there to increase stock of existing stock lines (if you were to return from an outbound shipment)